src/modules/Dashboard/components/FlowSummaryCard/FlowSummaryThumbnail/FlowSummaryThumbnail.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { Box, styled } from '@mui/material';
+
+import { theme, variables } from 'shared/styles';
+
+export const StyledRoot = styled(Box)({
+  background: variables.palette.primary_container,
+  borderRadius: 8,
+  display: 'flex',
+  flexShrink: 0,
+  flexWrap: 'wrap',
+  height: theme.spacing(8),
+  overflow: 'hidden',
+  placeContent: 'flex-start',
+  placeItems: 'flex-start',
+  position: 'relative',
+  width: theme.spacing(8),
```

Comment:
Nit: For reasons I haven't yet understood, I've seen PR review feedback requesting that only gaps & spacing to use `theme.spacing`, while width & height should be in `rem` units. 🤷🏻‍♂️ Again I don't know that this is documented anywhere (I think that's supposed to be an action item somewhere), just sharing what I've encountered.